src/FeatureFlags/OptionBasedFormEditor/Settings/General.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Give\FeatureFlags\OptionBasedFormEditor\Settings;
+
+/**
+ * @unreleased
+ */
+class General extends AbstractOptionBasedFormEditorSettings
+{
+    /**
+     * @unreleased
+     */
+    public function getDisabledOptionIds(): array
+    {
+        return [
+            //General Section
+            'success_page',
+            'failure_page',
+            // Currency Section
+            'auto_format_currency',
+            'currency_position',
+            'thousands_separator',
+            'decimal_separator',
+            'number_decimals',
+            'currency_preview',
```

Comment:
I think we can add `override_legacy_donation_management_pages` here as well.  I'm going to ask about the donation history shortcode https://givewp.com/documentation/core/shortcodes/donation_history/

Reply:
Resolved in 512ad4dc6a0e8c13cfb06b4553ddf969ab81e0fd